docs/guide.rst

```diff
@@ -1019,3 +1019,6 @@ Current Limits
 
 __ https://github.com/teemtee/tmt/labels/multihost
 __ https://github.com/teemtee/tmt/issues/2047
+
+
+.. include:: guide/testing-farm.inc.rst
```

Comment:
How about adding a "Test runners" header, plus a short paragraph, something like "tmt can run in many environments, your mileage may vary, and here's what we know about various runners", then followed by the very nice text about TF you already have? Then the note should point to this "test runners" header rather than directly to the TF section.

I would like to avoid TF seeming as the only option. The note says "yep, things may be different" and points to TF page directly, leaving no space for any other test runner. And I'd say there's at least one other already, our workstations when we develop tests, for example. I think it would be very helpful and very tidy if we could collect what we know about that runner as well as TF, put together a collection of quirks, things to look for, special setup to be aware of when running tmt (locally|in TF|in Github|...). And that's why "test runners" leading to TF directly feel like we would neglect any other test runner already, and I'd like to keep that space open for the future additions.

Reply:
Yeah, makes sense, done in 3f53cb1c.